src/Cart/Cart.jsx

```diff
@@ -0,0 +1,31 @@
+import { useStateContext } from "../context";
+import { CartItemCard } from "./CartItem-card";
+import { filterDataOnStatus } from "../Product-listing";
+import { CartValueDetails } from "./Cart-value";
+
+export const Cart = () => {
+  const { state } = useStateContext();
+  const dataToMap = filterDataOnStatus(state.itemsInCart);
+  return (
+    <>
+      <h1 className="text-center h6 page-title">My Cart</h1>
+      <div className="display-flex width-600px">
+        <ul className="column-60-pc list-style-none styled-list padding-1rem-borderbox">
+          {state.itemsInCart.length === 0 ? (
+            <h3 className="text-center">Cart is empty</h3>
+          ) : (
```

Comment:
Once I delete item from cart and its empty. It does not show this message that cart is empty. Can you check this please ?

Reply:
Yes, Fixed this.